src/formatDate.js

```diff
@@ -50,7 +50,27 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
```

Comment:
Prefer arrow functions over traditional function expressions for brevity and to avoid issues with this context. Convert the 'formatDate' function to an arrow function.